setup.py

```diff
@@ -6,7 +6,7 @@
     packages=['vent', 'vent.core', 'vent.core.file_drop',
               'vent.core.rq_worker', 'vent.core.rq_dashboard', 'vent.menus',
               'vent.core.rmq_es_connector', 'vent.helpers', 'vent.api'],
-    install_requires=['docker', 'npyscreen'],
+    install_requires=['docker', 'npyscreen', 'pyyaml'],
```

Comment:
this requirement also needs to be added line 97 in `dev/vendor/tinycore-python2/Dockerfile`